fire/SFMainMod.F90

```diff
@@ -448,7 +448,7 @@ subroutine  characteristics_of_crown ( currentSite, canopy_fuel_load, passive_cr
 
                 !sort crown fuel into bins from bottom to top of crown
                 !accumulate across cohorts to find density within canopy 1m sections
-                do ih = int(height_cbb), int(currentCohort%hite)
+                do ih = max(1, nint(height_cbb)), max(1, nint(currentCohort%hite))
                    biom_matrix(ih) = biom_matrix(ih) + crown_fuel_per_m
```

Comment:
This was needed to avoid attempting
`biom_matrix(0) = biom_matrix(0) + crown_fuel_per_m`